apps/filecoin-site/src/app/_components/Footer/NewsletterFormSchema.ts

```diff
@@ -0,0 +1,10 @@
+import { z } from 'zod'
```

Comment:
This can be refactored with `apps/ff-site/src/app/_components/NewsletterForm.tsx`

```js 
const NewsletterSchema = z.object({
  email: z
    .string()
    .min(1, { message: 'This field has to be filled.' })
    .email('This is not a valid email.'),
})
```